tests/test_ansible.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import unittest
+import shlex
+from helper import shell_process
+import re
+
+"""Moneo ansible single node test"""
+
+
+class AnsibleTestCase(unittest.TestCase):
+    """Ansible unit test class"""
+    
+    def check_docker_status(self):
+        """Helper to check docker container status"""
+        cmd='sudo docker container ls'
+        args = shlex.split(cmd)
+        result = shell_process.shell_cmd(args)
+        if ('grafana' in result) or ( 'prometheus' in result ):
+            return True #deployed
+        else:
+            return False #not deployed
+            
+    def check_exporter_status(self):
+        """Helper to check exporter status"""
+        cmds=['ps -f -C  python2','ps -f -C  python3']
```

Comment:
This is similar to the case for checking prometheus and grafana. Do you think it would be worth splitting the check for both exporters? It may make it easier to identify problems that may arise.

Reply:
This is more focused on ansible. in the next PR I have unit tests that cover the exporters exclusively. So coverage will be there.